fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/service/SavingsAccrualWritePlatformServiceImpl.java

```diff
@@ -158,13 +158,19 @@ private void addAccrualTransactions(SavingsAccount savingsAccount, final LocalDa
 
         final List<LocalDate> accrualTransactionDates = savingsAccount.retrieveOrderedAccrualTransactions().stream()
                 .map(transaction -> transaction.getTransactionDate()).toList();
+        final List<LocalDate> accrualTransactionDatesReverse = savingsAccount.retrieveOrderedAccrualTransactions().stream()
```

Comment:
Could you please rename this field? At first glance it seems to be a list of accrual transactions in reverse date order, but in fact it’s a list of reversed accrual transactions. The name should make that explicit.